packages/frontend-2/components/viewer/resources/VersionCard.vue

```diff
@@ -1,9 +1,10 @@
 <template>
   <button
     :class="`bg-foundation group relative block w-full space-y-2 rounded-md pb-2 text-left transition ${
-      clickable ? 'hover:bg-primary-muted' : 'cursor-default'
+      clickable
+        ? 'hover:bg-gray-100 dark:hover:bg-gray-700'
```

Comment:
non-semantic colors

Reply:
Non semantic colors can be okay at this stage if it looks the part (and we're missing a semantic color to do what it's needed here), and we can think of making them "official".